services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/profiili/ProfiiliData.kt

```diff
@@ -0,0 +1,9 @@
+package fi.hel.haitaton.hanke.profiili
+
+data class ProfiiliResponse(val data: ProfiiliData)
+
+data class ProfiiliData(val myProfile: MyProfile?)
+
+data class MyProfile(val verifiedPersonalInformation: Names?)
+
+data class Names(val firstName: String, val lastName: String, val givenName: String)
```

Comment:
The difference of firstName and givenName could be explained in a kdoc or something.

Reply:
I added documentation here.